docs/user-guide/computation.ipynb

```diff
@@ -102,8 +102,33 @@
    },
    "source": [
     "The implementation assumes uncorrelated data and is otherwise based on, e.g., [Wikipedia: Propagation of uncertainty](https://en.wikipedia.org/wiki/Propagation_of_uncertainty#Example_formulae).\n",
-    "See also [Propagation of uncertainties](../reference/error-propagation.rst) for the concrete equations used for error propagation.\n",
+    "See also [Propagation of uncertainties](../reference/error-propagation.rst) for the concrete equations used for error propagation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-warning\">\n",
+    "    <b>WARNING:</b>\n",
+    "\n",
+    "If an operand with variances is also [broadcast](#Broadcasting) in an operation then the resulting values will be correlated.\n",
+    "Scipp has no way of tracking or handling such correlations.\n",
+    "Subsequent operations that combine values of the result, such as computing the mean, will thus result in *underestimated uncertainties*.\n",
+    "Generally, the differences are negligible only if the variances of the broadcast operand are negligible.\n",
     "\n",
+    "Scipp's behavior in this case may change in the future.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
```

Comment:
```suggestion
   "metadata": { },
```
Does the jupyter cleanup tool not remove this kind of metadata?

Reply:
Maybe the new one does? Let me update the branch, it was just merged...

Reply:
By default it is not, but it could be configured to do so, specifically.